### DISCUSSION TITLE
Financiamento coletivo para Python Nordeste 2024

### DISCUSSION THREAD

**Discussion by allythy** [apyb/comunidade#113] [Geral]
O participante poderia contribuir financeiramente com organização da Python Nordeste 2024 fazendo uma doação de qualquer valor, mesmo que não pretenda vir ao evento. A doação seria feita para o Associação Python Brasil.

O valor arrecadado será usado no pagamento de despesas como:
- Confecção de materiais gráficos;
- Camisetas;
- Kits para os participantes;
- Coffee-breaks.

Seria possível fazer isso por meio da Associação Python Brasil?

**Comment by cecivieira** [comment]
Oi @allythy ,
não é possível a viabilização de recebimento de pequenas doações individuais diretamente na conta bancária da APyB. Nossa sugestão é que seja criado um financiamento coletivo ou adicionada uma opção de ingresso "doação" na plataforma de inscrição.

Lembrando que a Python Nordeste está sendo representada pela pessoa jurídica da APyB, então todo o seu caixa deve passar pelo nosso sistema contábil. Isso quer dizer que o direcionamento do montante arrecadado nas plataformas sugeridas deve ser a conta bancária da Associação.
